src/main/java/Duke.java

```diff
@@ -1,10 +1,134 @@
+            System.out.println("   " + tasks[currentID].toString());
+        }
+        else {
+            tasks[currentID].unmarkDone();
+            System.out.println("OK, I've marked this task as not done yet:");
+            System.out.println("   " + tasks[currentID].toString());
+        }
+        drawLine("_");
+    }
+
+    public static void printMenu() {
+        drawLine("_");
+        System.out.println("Try the following commands:");
+        System.out.println("1. list: show the list of tasks;");
+        System.out.println("2. todo: add a task without starting time/deadline to the list;");
+        System.out.println("3. deadline: add a task with deadline to the list;");
+        System.out.println("4. event: add a task with specific starting and ending time");
+        System.out.println("5, mark: mark a task as 'done' state;");
+        System.out.println("6. unmark: unmark a task from 'done' state;");
+        drawLine("_");
+    }
+
+    public static void doCommand(Task[] tasks, String text) {
+        String splittedText[] = text.split(" ", 2);
+        String command = splittedText[0];
+        String content = splittedText[splittedText.length - 1];
+        switch (command) {
+        case "list":
+            printTaskList(tasks);
+            break;
+        case "menu":
+            printMenu();
+            break;
+        case "mark":
+        case "unmark":
+            int currentID = Integer.parseInt(text.substring(text.length() - 1)) - 1;
+            boolean isDone;
+            if (text.contains("unmark")) {
+                isDone = false;
+                markTask(tasks, currentID, isDone);
+            }
+            else {
+                isDone = true;
+                markTask(tasks, currentID, isDone);
+            }
+            break;
+        case "todo":
+            Todo todoTask = new Todo(content, Task.getTaskCount());
+            tasks[Task.getTaskCount() - 1] = todoTask;
+            echo(tasks);
+            break;
+        case "deadline":
+            Deadline deadlineTask = new Deadline(content, Task.getTaskCount());
+            tasks[Task.getTaskCount() - 1] = deadlineTask;
+            echo(tasks);
+            break;
+        case "event":
+            Event eventTask = new Event(content, Task.getTaskCount());
+            tasks[Task.getTaskCount() - 1] = eventTask;
+            echo(tasks);
+            break;
+        default:
+            drawLine("_");
+            System.out.println("You have done nothing...?" + System.lineSeparator()
+                    + "if you need more instruction, please type 'menu'.");
+            drawLine("_");
+            break;
+        }
+    }
+
+    public static void loopCommand(Task[] tasks) {
+        Scanner in = new Scanner(System.in);
+        String text = in.nextLine();
+        while(!text.equals("bye")){
+            doCommand(tasks, text);
+            text = in.nextLine();
+        }
+    }
```

Comment:
Great to see that the names are verbs and the names makes the function of each method easy to understand